e2e/test/scenarios/visualizations-tabular/object_detail.cy.spec.js

```diff
@@ -163,11 +164,11 @@ describe("scenarios > question > object details", { tags: "@slow" }, () => {
 
     createQuestion(questionDetails, { visitQuestion: true });
 
-    // there should be a hover instead of click
-    // but realHover is flaky
-    cy.get("[data-testid=cell-data]").contains("4966277046676").click();
+    cy.findByRole("gridcell", { name: "4966277046676" }).should("exist"); //Check that cell is rendered before mouseing over
+    cy.findByRole("gridcell", { name: "4966277046676" }).trigger("mouseover");
```

Comment:
This test was flaking really hard. I believe the mouseover event was happening too quickly as the table was loading. This change ensures the element has rendered before we mouse over it, causing the button to appear